server/src/main/resources/application.yml

```diff
@@ -71,3 +71,16 @@ flyway:
   datasources:
     default:
       locations: classpath:db
+---
+service-account-credentials:
+  directory_id: ${ DIRECTORY_ID }
+  type: ${ TYPE }
+  project_id: ${ PROJECT_ID }
+  private_key_id: ${ PRIVATE_KEY_ID }
+  private_key: ${ PRIVATE_KEY }
+  client_email: ${ CLIENT_EMAIL }
+  client_id: ${ CLIENT_ID }
+  auth_uri: ${ AUTH_URI }
+  token_uri: ${ TOKEN_URI }
+  auth_provider_x509_cert_url: ${ AUTH_PROVIDER_X509_CERT_URL }
+  client_x509_cert_url: ${ CLIENT_X509_CERT_URL }
```

Comment:
I wonder if we shouldn't have just automagically mapped these from env variables like `SERVICE_ACCOUNT_DIRECTORY_ID`...those can be automagically brought in as params out of the environment without the need to add them to the yaml. Just a thought, we can discuss.

Reply:
I will try to refactor this as part of the bugfix issue.